api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -985,16 +980,16 @@ public OrderType getOrderTypeByConcept(Concept concept) {
 	public List<Concept> getTestSpecimenSources() {
 		return getSetMembersOfConceptSetFromGP(OpenmrsConstants.GP_TEST_SPECIMEN_SOURCES_CONCEPT_UUID);
 	}
-
+	
 	@Override
 	public Concept getNonCodedDrugConcept() {
 		String conceptUuid = Context.getAdministrationService().getGlobalProperty(OpenmrsConstants.GP_DRUG_ORDER_DRUG_OTHER);
-		if(org.apache.commons.lang.StringUtils.isNotBlank(conceptUuid)){
+		if (org.apache.commons.lang.StringUtils.isNotBlank(conceptUuid)) {
```

Comment:
Can you update this to use the StringUtils from spring since it is already imported in to the class, I think the equivalent method to call is StringUtils.hasText()